.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: ci
+on: [push, pull_request]
+
+jobs:
+  build_and_deploy:
+    env:
+        BASE_VERSION: 5.0.0.2012
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Download Nuget.exe
+      uses: NuGet/setup-nuget@v1.0.5
+      with:
+        nuget-version: 5.4.0
+    - name: setup-msbuild
+      uses: microsoft/setup-msbuild@v1
+    - name: Restore & Build
+      run: |
+        cd Xamarin.Forms.Platform.GTK
+        Nuget.exe restore Xamarin.Forms.Platform.GTK.csproj
+        msbuild Xamarin.Forms.Platform.GTK.csproj /property:Configuration=Release
+    - name: Package and upload
+      if: ${{ github.event_name == 'push' }}
```

Comment:
instead of this `if` here, I think it's better to not call nuget.exe push when branch is not master; this way the CI will still be run up to that process call even for pull requests (just in case they modify CI and they break it)

Reply:
I don't understand what you mean here the reason this Package and Upload step doesn't check if the branch is master is because default branches are 5.0.0 and etc here instead they package on "push" and not push on "pull_request"

Reply:
with `master` I was meaning the default branch, which in this case is 5.0.0

Reply:
(so re-read my comment now)